data_steward/bq_utils.py

```diff
@@ -536,6 +537,12 @@ def list_tables(dataset_id=None,
       for table in result['tables']:
           print table['id']
     """
+    warnings.warn(
```

Comment:
Maybe add a TODO here to remind us to remove this stuff in something like 6 months?

Reply:
I think I can do better- I created a ticket DC-1157 and listed the funcs to remove. Too many TODOs don't get done. :) I found other companies following this warning practice prior to removing library code.